docs/source/markdown/podman-pod-create.1.md

```diff
@@ -75,6 +75,13 @@ Set custom DNS options in the /etc/resolv.conf file that will be shared between
 
 Set custom DNS search domains in the /etc/resolv.conf file that will be shared between all containers in the pod.
 
+#### **--exit-policy**=*{continue,stop}*
```

Comment:
I know everyone ignores that but we have a https://github.com/containers/podman/blob/main/docs/MANPAGE_SYNTAX.md

and the correct syntax is
```suggestion
#### **--exit-policy**=**continue** | *stop*
```